lib/davinci_pas_test_kit/generated/v2.0.1/claim_inquiry/client_inquire_request_claim_inquiry_must_support_test.rb

```diff
@@ -68,7 +68,6 @@ class ClientInquireRequestClaimInquiryMustSupportTest < Inferno::Test
         * Claim.supportingInfo:DischargeDates.timing[x]
         * Claim.supportingInfo:PatientEvent.timing[x]
       )
-      verifies_requirements 'hl7.fhir.us.davinci-pas_2.0.1@36'
```

Comment:
This was flagged as a warning. becaise `36` is a server requirement and this is a client test. I just removed it, because none of the other requirements seemed to fit here.